nexus/src/db/lookup.rs

```diff
@@ -325,6 +325,31 @@ impl<'a> LookupPath<'a> {
         Sled { key: SledKey::PrimaryKey(Root { lookup_root: self }, id) }
     }
 
+    /// Select a resource of type SshKey, identified by its id
+    pub fn ssh_key_id(self, id: Uuid) -> SshKey<'a> {
+        SshKey { key: SshKeyKey::PrimaryKey(Root { lookup_root: self }, id) }
+    }
+
+    /// Select a resource of type SshKey, identified by its name
```

Comment:
I would not expect this function to exist on `LookupPath` because you can't look up an SshKey by its name alone -- you need a user, too.  There should already be a `SiloUser::ssh_key_name()` method by virtue of having put "SshKey" in the "children" list for the definition of "SiloUser" in this file.  So I think you can drop this function altogether and have callers just use `LookupPath(...).silo_user_id(opctx.authn.actor_required()?).ssh_key_name(name)`?

Reply:
Right, I think this was left over from when I was looking up all keys by ID and before I understood the auto-generated methods. Fixed in 3f4d9cd8.